src/factorial.js

```diff
@@ -0,0 +1,13 @@
+export default function factorial(num){
+  if(num > 0){
+    var array = [];
+    while(num > 0){
+      array.push(num);
+        num--;}
+    return array.reduce(function(a,b){return a*b;});
+  }else if(num === 0){
+    return 1;
+  }else{
+    return 'Invalid input';
+  }
+}
```

Comment:
Using spaces and newlines can make your code more readable.  Use your own judgement.  This is how I would do it:

```js
export default function factorial (num) {
  if (num > 0) {
    var array = [];
    while (num > 0) {
      array.push(num);
        num--;
    }
    return array.reduce(function (a, b) {return a * b;});
  } else if (num === 0) {
    return 1;
  } else {
    return 'Invalid input';
  }
}
```